src/components/@shared/FormFields/FilesInput/index.tsx

```diff
@@ -4,29 +4,23 @@ import { toast } from 'react-toastify'
 import FileInfo from './Info'
 import UrlInput from '../URLInput'
 import { InputProps } from '@shared/FormInput'
-import { getFileInfo } from '@utils/provider'
+import { fileInfo } from '@utils/provider'
 import { useWeb3 } from '@context/Web3'
 import { getOceanConfig } from '@utils/ocean'
-import { useCancelToken } from '@hooks/useCancelToken'
 import { initialValues } from 'src/components/Publish/_constants'
 
 export default function FilesInput(props: InputProps): ReactElement {
   const [field, meta, helpers] = useField(props.name)
   const [isLoading, setIsLoading] = useState(false)
   const { chainId } = useWeb3()
-  const newCancelToken = useCancelToken()
 
   function loadFileInfo(url: string) {
     const config = getOceanConfig(chainId || 1)
 
     async function validateUrl() {
       try {
         setIsLoading(true)
-        const checkedFile = await getFileInfo(
-          url,
-          config?.providerUri,
-          newCancelToken()
-        )
+        const checkedFile = await fileInfo(url, config?.providerUri)
```

Comment:
this would look like 
```
  const checkedFile = await  ProviderInstance.fileinfo(
        url,
        config.providerUri,
        (url: string, body: string) => {
          return axios.post(url, body, {
            cancelToken: newCancelToken()
          })
        }
      )
```

Reply:
Also the function will be renamed `fileinfo` - > `checkFileUrl` .
The `fileinfo` endpoint has multiple functionalities, so we will implement another function in ocean.js that will return the file info from a did (not relevant for this pr, just for context)